src/lib/server.ts

```diff
@@ -78,6 +78,13 @@ class Client {
         return;
       }
 
+      if (msg.command === "get_log_config") {
+        this.sendResultSuccess(msg.messageId, {
+          config: this.driver.getLogConfig(),
```

Comment:
Is everything returned from this function JSON serializable or should certain parts be manually converted (ie `Transport`)?

Reply:
See my comment here: https://github.com/zwave-js/zwave-js-server/pull/116#discussion_r577374393